.gitignore

```diff
@@ -4,3 +4,5 @@ profiler/*
 
 bin/
 pkg/
+
```

Comment:
Please remove it for the final commit

Reply:
it may be useful if someone uses GoLand or GoLand Plugin for the IntelliJ IDEA